src/components/controls/gamepad-controls.js

```diff
@@ -0,0 +1,110 @@
+var registerControls = require('../../core/controls').registerControls;
```

Comment:
In other words, I think we could also publish that separately as well because UX-wise, I'm not sure if I want to encourage that form of locomotion